.env

```diff
@@ -1 +1,6 @@
-DEVISE_JWT_SECRET_KEY=938f1fa2ad453f2b84760f0fc15425ac1554fd4ac23ae28f51642f4ae02fc82d1c27ccee216323f581561964347858bc665e2356364d3a810851a0c29c2d7d81
\ No newline at end of file
+DEVISE_JWT_SECRET_KEY=938f1fa2ad453f2b84760f0fc15425ac1554fd4ac23ae28f51642f4ae02fc82d1c27ccee216323f581561964347858bc665e2356364d3a810851a0c29c2d7d81
```

Comment:
Usually you don't want to check the .env file to git, since it may contain sensible information such as passwords or tokens